### ISSUE TITLE
Properly escape command arguments

### ISSUE THREAD

**Issue by ouuan** [FabijanZulj/blame.nvim#32]
Please use lists instead of strings for command arguments so that they are properly escaped.

PoC: `nvim '";notify-send injected"' +BlameToggle`

e.g.

```lua
    local blame_command = {"git", "--no-pager", "blame", "--line-porcelain",
        commit or "@",
        "--", filename,
    }
```

I suggest even those not vulnerable to command injection should also use lists.

**Comment by FabijanZulj** [comment]
Thanks! It was on my mind, I will fix it.

**Comment by FabijanZulj** [comment]
Should be fixed with #36
